recipes/gitlab-runner/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "gitlab-runner" %}
+{% set version = "12.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: "https://gitlab.com/gitlab-org/gitlab-runner.git"
```

You can replace it by
```suggestion
  git_url: "https://gitlab.com/gitlab-org/gitlab-runner/-/archive/v{{ version }}/gitlab-runner-v{{ version }}.tar.gz"
```

---

@marcelotrevisani, @xhochy, I am following the build procedure that is used for the semi-official [homebrew formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/gitlab-runner.rb), which includes inserting the git hash into the build output. I have no idea whether this is a requirement of the build, or just a useful piece of metadata when you are building from git.

This is why I have not used the tarball, but I am happy to receive advice on how to proceed.

---

I guess you can set the revision to whatever you like ... here it is set to conda-forge: https://github.com/conda-forge/fzf-feedstock/blob/master/recipe/meta.yaml 

This recipe also contains some different ideas building a go-recipe.

---

Revision is set to `deadbeef`